topics/admin/tutorials/customization/tutorial.md

```diff
@@ -0,0 +1,369 @@
+>       galaxy:
+>    +    # Branding
+>    +    brand: Mars 🚀
+>    +    logo_src: "https://training.galaxyproject.org/training-material/topics/admin/tutorials/customization/images/logo.png"
+>         # Main Configuration
+>         admin_users:
+>         - admin@example.org
+>    {% endraw %}
+>    ```
+>    {: data-commit="Add brand"}
+>
+> 1. Run the playbook.
+>
+>    > <code-in-title>Bash</code-in-title>
+>    > ```bash
+>    > ansible-playbook galaxy.yml
+>    > ```
+>    > {: data-cmd="true"}
+>    {: .code-in}
+>
+>    ![screenshot of fictional "Galaxy Mars" start page, with the brand text set to "Mars"](images/galaxy-mars-brand.png "Your Galaxy start page should now look something like this")
+{: .hands_on}
+
+## Custom Welcome Page
+
+The welcome page is an html document embedded in Galaxy's start page.
+It is what users see when first navigating to your instance.
+This page can be used to communicate what your instance is about, and share news and updates with your users.
+
+> <hands-on-title>Creating a custom welcome page</hands-on-title>
+>
+> 1. Set the location of your welcome page in `group_vars/galaxyservers.yml`
+> 
+>    Under `galaxy_config.galaxy` specify the location of your welcome page
+>
+>    {% raw %}
+>    ```diff
+>    --- a/group_vars/galaxyservers.yml
+>    +++ b/group_vars/galaxyservers.yml
+>    @@ -86,6 +86,11 @@ galaxy_config:
+>               - job-handlers
+>               - workflow-schedulers
+>     
+>    +galaxy_config_files:
+>    +  - src: files/galaxy/welcome.html
+>    +    dest: "{{ galaxy_mutable_config_dir }}/welcome.html"
+>    +    mode: "0755"
+>    +
+>     # Certbot
+>     certbot_auto_renew_hour: "{{ 23 |random(seed=inventory_hostname)  }}"
+>     certbot_auto_renew_minute: "{{ 59 |random(seed=inventory_hostname)  }}"
+>    {% endraw %}
+>    ```
+>    {: data-commit="Add welcome url to the config"}
+>
+> 2. Let's then develop the associated template, add a title and some text to our welcome page:
+>
+>    {% raw %}
+>    ```diff
+>    --- /dev/null
+>    +++ b/files/galaxy/welcome.html
+>    @@ -0,0 +1,11 @@
+>    +<!DOCTYPE html>
+>    +<html lang="en">
+>    +    <head>
+>    +        <meta charset="utf-8">
+>    +        <link href="/static/dist/base.css" rel="stylesheet" type="text/css" />
+>    +    </head>
+>    +    <body>
+>    +        <h1>Welcome to the Galaxy Mars instance!</h1>
+>    +        <p>The only Galaxy instance on mars.</p>
+>    +    </body>
+>    +</html>
+>    diff --git a/group_vars/galaxyservers.yml b/group_vars/galaxyservers.yml
+>    index 3d7fd0e..8d8bd40 100644
+>    --- a/group_vars/galaxyservers.yml
+>    +++ b/group_vars/galaxyservers.yml
+>    @@ -86,10 +86,9 @@ galaxy_config:
+>               - job-handlers
+>               - workflow-schedulers
+>     
+>    -galaxy_config_files:
+>    +galaxy_config_files_public:
+>       - src: files/galaxy/welcome.html
+>         dest: "{{ galaxy_mutable_config_dir }}/welcome.html"
+>    -    mode: "0755"
```

Comment:
```suggestion
>    -galaxy_config_files:
>    +galaxy_config_files_public:
>       - src: files/galaxy/welcome.html
>         dest: "{{ galaxy_mutable_config_dir }}/welcome.html"
>    -    mode: "0755"
```
I think this diff and the one in the previous step are off?

Reply:
Fixed, great catch.